db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2024_01_09_165800) do
+ActiveRecord::Schema[7.2].define(version: 2024_01_09_165800) do
```

Comment:
We should rollback this change, just to make sure nothing else is updated unintentionally here